components/gitpod-db/go/cost_center.go

```diff
@@ -262,31 +262,36 @@ func (c *CostCenterManager) ListLatestCostCentersWithBillingTimeBefore(ctx conte
 	return results, nil
 }
 
-func (c *CostCenterManager) ResetUsage(ctx context.Context, cc CostCenter) (CostCenter, error) {
+func (c *CostCenterManager) ResetUsage(ctx context.Context, id AttributionID) (CostCenter, error) {
+	logger := log.WithField("attributionId", id)
+	now := time.Now().UTC()
+	cc, err := getCostCenter(ctx, c.conn, id)
+	if err != nil {
+		return cc, err
+	}
 	if cc.BillingStrategy != CostCenter_Other {
 		return CostCenter{}, fmt.Errorf("cannot reset usage for Billing Strategy %s for Cost Center ID: %s", cc.BillingStrategy, cc.ID)
 	}
+	if !cc.IsExpired() {
+		logger.Info("Skipping ResetUsage because next billing cycle is in the future.")
+		return cc, nil
+	}
 
-	now := time.Now().UTC()
-
+	logger.Info("Running `ResetUsage`.")
```

Comment:
```suggestion
	logger.WithField("attribution_id", id).WithField("cost_center", cc).Infof("Resetting usage for Attribution ID %s", id)
```

Reply:
👍  Added the cost center. the attributiond id was already included and I like to keep the message static, as it's a bit easier to filter on that way.